content/documentation/install-uninstall.md

```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
+    You will be prompted to enter your computer password. Press enter. You should receive a message that PrinceXML will be installed in the `/usr/local` directory. Press enter again. If successful you will see a message in the Terminal saying "installation complete." Or, if you get a "no such file or directory" message after the first line, the file you dowloaded is either not in your Downloads folder, or is not named "prince". Correct as necessary and try again.
 
-    When complete, type `cd` into the Terminal to return to your home directory.
+    When complete, type `cd` into the Terminal to return to your {{< q-def "home directory">}}.
 
     ```text
     cd
     ```
 
-5. The **Quire CLI** ({{< q-def "command-line interface" >}}) is the core of Quire. It is used for creating, previewing, and outputting projects. You can learn more about the Quire CLI in the [*Quire CLI Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into your Terminal to download the Quire CLI to your computer from GitHub.
+6. **Quire** is operated through a {{< q-def "command-line interface" >}} (CLI) that enables you to create, preview and output publications using Terminal with commands like `quire new`, `quire preview` and `quire site`. You can learn more in the [*Quire Commands*](/documentation/quire-cli/) chapter of our guide. Copy and paste the following line into Terminal to install Quire:
 
     ```text
-    git clone https://github.com/gettypubs/quire-cli.git
+    npm install --global @thegetty/quire-cli
     ```
 
-    You may be asked to enter your GitHub username and password. When complete, install the CLI by copying and pasting the following commands into your Terminal and pressing enter. The first resets user permissions for your local directory, the second installs Quire.
+    The Quire installation process may take several minutes, during which time there will be considerable messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation.
 
-    ```text
-    sudo chown -R $USER /usr/local
-    ```
+    To verify installation, enter the command below. This will give you a list of [commands](/documentation/quire-cli/) that will help you get started using and navigating Quire. And if you get "command not found" it means it was not installed correctly.
 
     ```text
-    cd quire-cli
-    npm install -g
+    quire --help
     ```
 
-When complete, type `quire --version` to confirm proper installation. It should return a version number, otherwise, if it says `command not found`, the Quire CLI has not been properly installed. Refer to the [*Troubleshooting*](#troubleshooting) section below.
+    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```

Comment:
We should also add the link to the Troubleshooting section of the Forum once that is up and running. I can just make a note to add this in later.